components/search_hint/search_hint.tsx

```diff
@@ -46,14 +46,20 @@ const SearchHint = (props: Props): JSX.Element => {
                         />
                     </div>
                     <div className='button-container'>
-                        <button onClick={() => props.onSearchTypeSelected && props.onSearchTypeSelected('messages')}>
+                        <button
+                            className={props.highlightedIndex === 0 ? 'highlighted' : ''}
```

Comment:
aren't we using something like `clsx` in the webapp?

In this case it is not really needed, but when combining multiple conditional classname assignments it really helps a lot.

Just a suggestion to look into 👾

Reply:
Done, thanks!